test/run/build-rpms.bats

```diff
@@ -54,4 +74,63 @@ load ../common
     echo "$output"
     [[ $status -eq 0 ]]
     [[ $output = '' ]]
+    rm -rf "${BATS_TMPDIR}/rpmbuild"
+}
+
+@test 'build/install epel8 RPMS' {
+    prerequisites_ok centos8
+    img=${ch_imgdir}/centos8
+    image_ok "$img"
+    if [[ -d ${BATS_TMPDIR}/rpmbuild/SOURCES/charliecloud ]]; then
+        rm -rf "${BATS_TMPDIR}/rpmbuild/SOURCES/charliecloud"
+    fi
+
+    # Build and install RPMs into CentOS 8 image.
+    (cd .. && packaging/fedora/build --install --image="$img" \
+                                     --rpmbuild="$BATS_TMPDIR/rpmbuild" HEAD)
+}
+
+@test 'check epel8 RPM files' {
```

Comment:
Are these two tests close enough that they could be put into functions and defined once?

Reply:
Not really. I tried and ended up with a function that had if statements to avoid checking for the charliecloud-test package on el8.